src/components/startscreen/LoadingScreen.tsx

```diff
@@ -8,21 +8,7 @@ type LoadingScreenProps = {
 };
 
 export const LoadingScreen = ({ setScene }: LoadingScreenProps) => {
-	const { progress } = useProgress();
-	const button = document.querySelector('.loadingScreen__button') as HTMLElement;
-	if (button) {
-		if (progress < 100) {
-			button.style.backgroundColor = 'grey';
-		}
-		if (progress === 100) {
-			button.style.backgroundColor = 'rgba(0, 149, 7, 0.729)';
-		}
-	}
-	function showOverworld() {
-		if (progress === 100) {
-			setScene(Scene.Overworld);
-		}
-	}
+
 	return (
 		<>
 			<div onClick={showOverworld} onTouchStart={showOverworld} className="loadingScreen">
```

Comment:
Where is function showOverworld?

Reply:
logic made it simpler -> not needed as function